Source/WebCore/Modules/compression/DecompressionStreamDecoder.cpp

```diff
@@ -34,11 +34,10 @@ namespace WebCore {
 
 ExceptionOr<RefPtr<Uint8Array>> DecompressionStreamDecoder::decode(const BufferSource&& input)
 {
-    auto* data = input.data();
-    if (!data)
+    if (!input.data())
         return Exception { ExceptionCode::TypeError, "No data provided"_s };
```

Comment:
Is this really a reachable code path? Do we have test coverage? I’m kind of surprised that we have a “no data” case distinct from “data length zero”.